lib/core/board.rb

```diff
@@ -33,30 +56,30 @@ def diagonal
   def anti_diagonal
     tiles = []
     position = @size - 1
-    until position > (@size*@size - @size)
-      tiles << @simplified_board[position]
+    until position > (total_tiles - @size)
```

Comment:
And be consistent, why no comment on this one? Inevitably, you'll delete the other method leaving any perceived benefit of that comment to be lost.